cloud/pkg/apis/edgeclusters/v1/types.go

```diff
@@ -98,11 +98,11 @@ type EdgeClusterSpec struct {
 	Labels map[string]string `json:"labels,omitempty"`
 }
 
-// EdgeClusterStatus is a description of Mission status
-type EdgeClusterStatus struct {
+// EdgeClusterState is a description of Mission status
```

Comment:
status or state?

Reply:
I think variable is named State but it's really status.

Reply:
fixed. It is 'state' now.